vortex-array/src/arrays/chunked/compute/elementwise.rs

```diff
@@ -0,0 +1,48 @@
+use vortex_error::{VortexExpect, VortexResult};
+
+use crate::Array;
+use crate::arrays::ChunkedArray;
+use crate::compute::{ComputeFn, InvocationArgs, Output, slice};
+
+impl ChunkedArray {
+    /// Invoke an element-wise compute function over a chunked array.
+    pub(in crate::arrays::chunked) fn invoke_elementwise(
+        &self,
+        compute_fn: &ComputeFn,
+        args: &InvocationArgs,
+    ) -> VortexResult<Option<Output>> {
+        assert!(compute_fn.is_elementwise());
+
+        // We know the first input argument is ourselves, if there is one other input argument
+        // then we can delegate the compute function. Otherwise, we return None.
+        if args.inputs.len() != 2 || args.inputs[1].array().is_none() {
+            return Ok(None);
+        }
+        let rhs = args.inputs[1].array().vortex_expect("checked already");
```

Comment:
See, we sort of assume that compute functions with two inputs are executed in this way.

For example, `is_in` has two inputs, but the RHS might be an arbitrary length list. In theory it's still element-wise, but we can't slice the RHS in the same way that we're doing in this function...